app/helpers/application_helper.rb

```diff
@@ -10,8 +10,7 @@ def present(object, klass = nil)
   def user_button_display
     button_array = []
     if current_user
-      button_array << link_to("Logout", logout_path, method: :delete)
-      button_array << current_admin_display
+      # button_array << current_admin_display
```

Comment:
Can we discuss this later? Looks like we are currently killing the button_array used for navigation. Sounds fine, but I want to make sure we all understand how this works going forward, or if we need to adjust to a different strategy. 😄